app/assets/stylesheets/application.scss

```diff
@@ -0,0 +1,52 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+h1,h2,h3,h4 {
+	color: #24C73A;
```

Comment:
Color literals like `#24C73A` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#24C73A` should be written as `#24c73a`<br>Line should be indented with spaces, not tabs<br>Each selector in a comma sequence should be on its own line